copter/source/docs/terrain-following-manual-modes.rst

```diff
@@ -4,7 +4,7 @@
 Surface Tracking
 ================
 
-Copter supports low altitude Surface Tracking in nearly all modes including pilot controlled modes like :ref:`AltHold <altholdmode>`, :ref:`Loiter <loiter-mode>` and :ref:`PosHold <poshold-mode>` and also fully autonomous modes including :ref:`AUTO <auto-mode>` and :ref:`Guided <ac2_guidedmode>`.
+Copter supports low altitude Surface Tracking, either a floor/ground or a ceiling, in nearly all modes including pilot controlled modes like :ref:`AltHold <altholdmode>`, :ref:`Loiter <loiter-mode>` and :ref:`PosHold <poshold-mode>` and also fully autonomous modes including :ref:`AUTO <auto-mode>` and :ref:`Guided <ac2_guidedmode>`.
```

Comment:
The new additions are fine but the original wording should be corrected to clarify that "Surface Tracking" only applies to pilot controlled modes.  So this part should be removed at least:

> and also fully autonomous modes including :ref:`AUTO <auto-mode>` and :ref:`Guided <ac2_guidedmode>`